_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -389,7 +411,7 @@ These two HTML `a` elements have the same [accessible name][] and link to the [s
 </html>
 ```
 
-#### Inapplicable Example 6
+#### Inapplicable Example 7
 
 These two HTML `a` elements have the same [accessible name][] but different [programmatically determined link contexts][programmatically determined link context] because the `div` elements place them in different display blocks.
```

Comment:
```suggestion
These two HTML `a` elements have the same [accessible name][], the same [accessible description][], but different [programmatically determined link contexts][programmatically determined link context] because the `div` elements place them in different display blocks.
```